main.rb

```diff
@@ -0,0 +1,20 @@
+def foo
+  eval(foo)
+end
```

Comment:
__[staging][rubocop]__
* The use of `eval` is a serious security risk.

<!-- Comment by [SideCI](https://sideci.com) -->